Error G5FE39F1E: Type '$DeviceInfoCopyWith' not found, G297C951C: Can't declare a member that conflicts with an inherited one.

@zeshuaro This is not fixed in `2.3.6` and also not in `2.4.0` When i use: `device_preview_plus: 2.3.6` in my pubspec.yaml. I still het the folowing error:

```
/C:/Users/**/AppData/Local/Pub/Cache/hosted/pub.dev/device_preview_plus-2.3.6/lib/src/state/custom_device.dart(29,3): error G5FE39F1E: Type '$DeviceInfoCopyWith' not found. [C:\repo\applications\lisatruck\build\windows\x64\flutter\flutter_assemble.vcxproj]
/C:/Users/**/AppData/Local/Pub/Cache/hosted/pub.dev/device_preview_plus-2.3.6/lib/src/state/custom_device.dart(29,39): error G297C951C: Can't declare a member that conflicts with an inherited one. [C:\repo\applications\lisatruck\build\windows\x64\flutter\flutter_assemble.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Microsoft\VC\v170\Microsoft.CppCommon.targets(254,5): error MSB8066: Custom build for 'C:\repo\applications\lisatruck\build\windows\x64\CMakeFiles\bbdbfa2f2471cd288c9d8da58b593cc9\flutter_windows.dll.rule;C:\repo\applications\lisatruck\build\windows\x64\CMakeFiles\c108df03244f07d198ece8e9d44ec0b5\flutter_assemble.rule' exited with code 1. [C:\repo\applications\lisatruck\build\windows\x64\flutter\flutter_assemble.vcxproj]
Error: Build process failed.
```

Version: `2.4.0`
```
/C:/Users/**/AppData/Local/Pub/Cache/hosted/pub.dev/device_preview_plus-2.4.0/lib/src/state/custom_device.dart(29,3): error G5FE39F1E: Type '$DeviceInfoCopyWith' not found. [C:\repo\applications\lisatruck\build\windows\x64\flutter\flutter_assemble.vcxproj]
/C:/Users/**/AppData/Local/Pub/Cache/hosted/pub.dev/device_preview_plus-2.4.0/lib/src/state/custom_device.dart(29,39): error G297C951C: Can't declare a member that conflicts with an inherited one. [C:\repo\applications\lisatruck\build\windows\x64\flutter\flutter_assemble.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Microsoft\VC\v170\Microsoft.CppCommon.targets(254,5): error MSB8066: Custom build for 'C:\repo\applications\lisatruck\build\windows\x64\CMakeFiles\bbdbfa2f2471cd288c9d8da58b593cc9\flutter_windows.dll.rule;C:\repo\applications\lisatruck\build\windows\x64\CMakeFiles\c108df03244f07d198ece8e9d44ec0b5\flutter_assemble.rule' exited with code 1. [C:\repo\applications\lisatruck\build\windows\x64\flutter\flutter_assemble.vcxproj]
Error: Build process failed.
```

I'm on flutter 3.32.0 and did flutter clean, and removed puspec.lock

<details>
<summary>This is the contents of this file: device_preview_plus 2.4.0/lib/src/state/custom_device.dart(29,3)</summary>

```dart
import 'package:device_preview_plus/device_preview_plus.dart';
import 'package:flutter/foundation.dart';
import 'package:flutter/material.dart';

/// A device info that can be customized at runtime to update
/// its current preview.
///
/// See also:
///
/// * [DeviceInfo] which describes a simulated device.
class CustomDeviceInfo implements DeviceInfo {
  /// Creates a new custom device.
  const CustomDeviceInfo._({required this.identifier, required this.data});

  /// Create a new custom device from stored [data].
  factory CustomDeviceInfo(CustomDeviceInfoData data) {
    return CustomDeviceInfo._(
      data: data,
      identifier: CustomDeviceIdentifier(data),
    );
  }

  static const _painter = GenericTabletFramePainter();

  /// The data that contains all of the custom device properties.
  final CustomDeviceInfoData data;

  @override
  $DeviceInfoCopyWith<DeviceInfo> get copyWith =>
      throw UnsupportedError('Read only');

  @override
  final DeviceIdentifier identifier;

  @override
  String get name => data.name;

  @override
  double get pixelRatio => data.pixelRatio;

  @override
  EdgeInsets? get rotatedSafeAreas => null;

  @override
  EdgeInsets get safeAreas => data.safeAreas;

  @override
  Size get screenSize => data.screenSize;

  @override
  Path get screenPath => _painter.createScreenPath(screenSize);

  @override
  Size get frameSize => _painter.calculateFrameSize(screenSize);

  @override
  CustomPainter get framePainter => _painter;

  @override
  List<DiagnosticsNode> debugDescribeChildren() => [];

  @override
  void debugFillProperties(DiagnosticPropertiesBuilder properties) {
    properties
      ..add(StringProperty('name', name))
      ..add(DoubleProperty('pixelRatio', pixelRatio))
      ..add(DiagnosticsProperty('safeAreas', safeAreas))
      ..add(DiagnosticsProperty('screenSize', screenSize))
      ..add(DiagnosticsProperty('frameSize', frameSize))
      ..add(DiagnosticsProperty('identifier', identifier));
  }

  @override
  DiagnosticsNode toDiagnosticsNode({
    String? name,
    DiagnosticsTreeStyle? style,
  }) {
    return DiagnosticsProperty<DeviceInfo>(
      name ?? 'CustomDeviceInfo',
      this,
      style: style ?? DiagnosticsTreeStyle.dense,
    );
  }

  @override
  String toStringDeep({
    String prefixLineOne = '',
    String? prefixOtherLines,
    DiagnosticLevel minLevel = DiagnosticLevel.debug,
    int wrapWidth = 65,
  }) {
    return toDiagnosticsNode().toStringDeep(
      prefixLineOne: prefixLineOne,
      prefixOtherLines: prefixOtherLines,
      minLevel: minLevel,
      wrapWidth: wrapWidth,
    );
  }

  @override
  String toStringShallow({
    String joiner = ', ',
    DiagnosticLevel minLevel = DiagnosticLevel.debug,
  }) {
    final properties = <String>[
      'name: $name',
      'pixelRatio: $pixelRatio',
      'safeAreas: $safeAreas',
      'screenSize: $screenSize',
      'frameSize: $frameSize',
    ];
    return properties.join(joiner);
  }

  @override
  String toStringShort() {
    return name;
  }

  @override
  String toString({DiagnosticLevel minLevel = DiagnosticLevel.info}) {
    return 'DeviceInfo(identifier: $identifier, name: $name, rotatedSafeAreas: $rotatedSafeAreas, '
        'safeAreas: $safeAreas, screenPath: $screenPath, pixelRatio: $pixelRatio, '
        'framePainter: $framePainter, frameSize: $frameSize, screenSize: $screenSize)';
  }
}

/// The device identifier for a [CustomDeviceInfo].
class CustomDeviceIdentifier implements DeviceIdentifier {
  /// Create a new custom device identifier from the given [data].
  const CustomDeviceIdentifier(this.data);

  /// The custom device data that this identifier represents.
  final CustomDeviceInfoData data;

  /// The predefined key associated to the custom device.
  static const identifier = 'custom_device';

  @override
  String get name => data.name;

  @override
  TargetPlatform get platform => data.platform;

  @override
  DeviceType get type => data.type;

  @override
  // ignore: non_nullable_equals_parameter
  bool operator ==(dynamic other) {
    return identical(this, other) ||
        (other is DeviceIdentifier &&
            other.name == name &&
            other.type == type &&
            other.platform == platform);
  }

  @override
  int get hashCode =>
      runtimeType.hashCode ^ name.hashCode ^ type.hashCode ^ platform.hashCode;
}
```
</details>

--- a/lib/src/state/custom_device.dart
+++ b/lib/src/state/custom_device.dart
@@ -1,5 +1,4 @@
 import 'package:device_preview_plus/device_preview_plus.dart';
-import 'package:flutter/foundation.dart';
 import 'package:flutter/material.dart';
 
 /// A device info that can be customized at runtime to update
@@ -26,8 +25,19 @@ class CustomDeviceInfo implements DeviceInfo {
   final CustomDeviceInfoData data;
 
   @override
-  $DeviceInfoCopyWith<DeviceInfo> get copyWith =>
-      throw UnsupportedError('Read only');
+  CustomDeviceInfo copyWith({
+    DeviceIdentifier? identifier,
+    String? name,
+    EdgeInsets? rotatedSafeAreas,
+    EdgeInsets? safeAreas,
+    Path? screenPath,
+    double? pixelRatio,
+    CustomPainter? framePainter,
+    Size? frameSize,
+    Size? screenSize,
+  }) {
+    throw UnsupportedError('Read only');
+  }
 
   @override
   final DeviceIdentifier identifier;
@@ -55,67 +65,6 @@ class CustomDeviceInfo implements DeviceInfo {
 
   @override
   CustomPainter get framePainter => _painter;
-
-  @override
-  List<DiagnosticsNode> debugDescribeChildren() => [];
-
-  @override
-  void debugFillProperties(DiagnosticPropertiesBuilder properties) {
-    properties
-      ..add(StringProperty('name', name))
-      ..add(DoubleProperty('pixelRatio', pixelRatio))
-      ..add(DiagnosticsProperty('safeAreas', safeAreas))
-      ..add(DiagnosticsProperty('screenSize', screenSize))
-      ..add(DiagnosticsProperty('frameSize', frameSize))
-      ..add(DiagnosticsProperty('identifier', identifier));
-  }
-
-  @override
-  DiagnosticsNode toDiagnosticsNode({
-    String? name,
-    DiagnosticsTreeStyle? style,
-  }) {
-    return DiagnosticsProperty<DeviceInfo>(
-      name ?? 'CustomDeviceInfo',
-      this,
-      style: style ?? DiagnosticsTreeStyle.dense,
-    );
-  }
-
-  @override
-  String toStringDeep({
-    String prefixLineOne = '',
-    String? prefixOtherLines,
-    DiagnosticLevel minLevel = DiagnosticLevel.debug,
-    int wrapWidth = 65,
-  }) {
-    return toDiagnosticsNode().toStringDeep(
-      prefixLineOne: prefixLineOne,
-      prefixOtherLines: prefixOtherLines,
-      minLevel: minLevel,
-      wrapWidth: wrapWidth,
-    );
-  }
-
-  @override
-  String toStringShallow({
-    String joiner = ', ',
-    DiagnosticLevel minLevel = DiagnosticLevel.debug,
-  }) {
-    final properties = <String>[
-      'name: $name',
-      'pixelRatio: $pixelRatio',
-      'safeAreas: $safeAreas',
-      'screenSize: $screenSize',
-      'frameSize: $frameSize',
-    ];
-    return properties.join(joiner);
-  }
-
-  @override
-  String toStringShort() {
-    return name;
-  }
 
   @override
   String toString({DiagnosticLevel minLevel = DiagnosticLevel.info}) {

--- a/pubspec.yaml
+++ b/pubspec.yaml
@@ -13,7 +13,7 @@ dependencies:
   flutter_localizations:
     sdk: flutter
   provider: ^6.1.5
-  device_frame_plus: ^1.3.1
+  device_frame_plus: ^1.4.0
   freezed_annotation: ^3.0.0
   json_annotation: ^4.9.0
   shared_preferences: ^2.5.3

